package.json

```
@@ -267,6 +267,11 @@
         "command": "ocaml.search-by-type",
         "category": "OCaml",
         "title": "Search a value by type or polarity"
+      },
+      {
+        "command": "ocaml.navigate-typed-holes",
+        "category": "OCaml",
+        "title": "List typed holes in the file for navigation."
```

to have the same format as other titles:

```suggestion
        "title": "List typed holes in the file for navigation"
```